tests/test_cldice_loss.py

```diff
@@ -0,0 +1,56 @@
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import unittest
+
+import numpy as np
+import torch
+from parameterized import parameterized
+
+from monai.losses import SoftclDiceLoss, SoftDiceclDiceLoss
+
+TEST_CASES = [
+    [  # shape: (1, 4), (1, 4)
+        {"y_pred": torch.ones((100, 3, 256, 256)), "y_true": torch.ones((100, 3, 256, 256))},
```

Comment:
hi @SauravMaheshkar could you please help make these test cases smaller? the github test runner doesn't seem to have enough memory for these https://github.com/Project-MONAI/MONAI/actions/runs/5655050610/job/15327390836

Reply:
Okay will try

Reply:
If that fixes the OOM error then it LGTM 😄